README.md

```diff
@@ -15,7 +15,8 @@ To install, follow the detailed installation instructions at http://wiki.volksza
 If you're impatient you can quickstart using (Debian Bullseye or Ubuntu 18.04 LTS):
 
     sudo apt-get install build-essential git-core cmake pkg-config subversion libcurl3-dev \
-      libgnutls-dev libsasl2-dev uuid-dev uuid-runtime libtool dh-autoreconf libunistring-dev
+      libgnutls-dev libsasl2-dev uuid-dev uuid-runtime libtool dh-autoreconf libunistring-dev \
+      libcurl4-gnutls-dev libjansson-dev libmosquitto-dev micro-httpd gnutls-dev libmicrohttpd-dev
```

Comment:
are you SURE on `libjansson-dev`? vzlogger uses libjson, so i think that should be `libjson-c-dev`...?
and i'd prefer `libcurl-(ssl-)dev`, unless we really need gnutls? (there was something about licensing with openssl...)
also, mosquitto (mqtt) and microhttpd (builtin webserver) are optional features, so i'd list them separately.